src/components/side_panel/conditional_formatting/cf_editor/cf_editor.ts

```diff
@@ -200,7 +195,7 @@ export class ConditionalFormattingEditor extends Component<Props, SpreadsheetChi
   icons = ICONS;
   iconSets = ICON_SETS;
   getTextDecoration = getTextDecoration;
-  colorNumberString = colorNumberString;
+  colorNumberString = colorNumberToHex;
```

Comment:
```suggestion
  colorNumberToHex = colorNumberToHex;
```
to keep consistency. Only usage is in `data_bar_rule_editor.xml`